### PR TITLE
sort missing files. closes #486

### LINKED ISSUE #486: List remaining files in alphabetical order in XML batch upload
**STORY**
As an administrator, I would like the list of files I need to find in XML upload to be sorted in alphabetical order, so that it's easier to match them up in my file browser.  (see screenshot of current behavior below)

> *NOTE:* this is a pretty low priority, but would be a nice enhancement if we're ever working on this section of the codebase.

mira-486 alpha sort list of files for upload

### PR DESCRIPTION
**Pull request by tonyc**

### CODE REVIEW

File: app/views/batch/xml_imports/edit.html.erb [hunk 1]
```diff
@@ -14,9 +14,9 @@
 <% if missing = @batch.missing_files %>
   <div class="well">
     The following files in this batch still need to be uploaded:<br/>
-    <ul>
-      <% missing.each do |file| %>
-        <li><%= file %></li>
+    <ul class="missing_files">
+      <% missing.sort_by { |a| [a.upcase, a] }.each_with_index do |file, index| %>
```
Comment (jcoyne) [inline app/views/batch/xml_imports/edit.html.erb]: can this code get moved into `@batch.missing_files`?

File: app/views/batch/xml_imports/edit.html.erb [hunk 2]
```diff
@@ -14,9 +14,9 @@
...
+        <li class="missing_file_<%= index %>"><%= file %></li>
```
Comment (jcoyne) [inline app/views/batch/xml_imports/edit.html.erb]: why are we adding this class. You can run the tests with `:nth-child()`

File: app/views/batch/xml_imports/edit.html.erb (hunk 1 above)
Comment (tonyc) [inline app/views/batch/xml_imports/edit.html.erb]: It seems like presentation logic for Mira. Should it get moved to BatchXmlImport in tufts_models?

Comment (jcoyne) [inline app/views/batch/xml_imports/edit.html.erb]: Eh, probably not.

**Pull request merged**